Azure/azure-cli
# [Security] Introducing securitySolutionsReferenceData CLI

**Description**  
[Security] Add securitySolutionsReferenceData commands

Add securitySolutionsReferenceData commands, helps you control which applications can run on your Azure and non-Azure machines (Windows and Linux)

**Testing Guide**  
The commands have been tested by running this test function "test_security_securitySolutionsReferenceData"

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [*] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [*] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).
